pkg/tanka/parse_test.go

```diff
@@ -3,42 +3,30 @@ package tanka
 import (
```

Comment:
Since `pkg/tanka/parse.go` was removed, does it make sense for this file to still be called `pkg/tanka/parse_test.go`?